rustls-libssl/MATRIX.md

```diff
@@ -452,7 +452,7 @@
 | `SSL_set_psk_server_callback` [^psk] |  |  |  |
 | `SSL_set_psk_use_session_callback`  |  |  |  |
 | `SSL_set_purpose`  |  |  |  |
-| `SSL_set_quiet_shutdown`  |  | :white_check_mark: |  |
+| `SSL_set_quiet_shutdown`  |  | :white_check_mark: | :white_check_mark: |
```

Comment:
Is this something we _should_ implement? Or would it be OK to make it a no-op stub?

I'm not sure I understand the use case and it isn't something that Rustls supports natively.

Reply:
nginx uses this when it detects the underlying connection has gone away, and it doesn't want to send a `close_notify` alert into a broken socket when the connection is being torn down. But nginx still calls `SSL_shutdown` during tear down, so this means we do IO for those calls.

Reply:
Thanks for the contextual information. That makes sense :+1: